### DISCUSSION TITLE
More sort systems

### DISCUSSION THREAD

**Discussion by gouglov** [PaulWoitaschek/Voice#3169] [Ideas]
### Overview

I don't understand how my ebooks are sorted. I only notice that the folders name is not relevant for sorting. Can we hope for a option in order to change the sorting system ?

### How to reproduce

Just open the app and see how the ebooks are sorted.

### Version

8.1.3

### Android Version

13

### Device Model

Pixel 4a

**Comment by JtxGit** [comment]
I had exactly the same reaction after opening this app the first time (I could not figure out any logical reason behind the default book sorting)... Would be great to have some way to sort the books according to user preferences that are more logical.
